flexget/plugins/sites/serienjunkies.py

```diff
@@ -166,7 +166,7 @@ def find_all_titles(self, search_title):
     def check_language(self, languages):
         # Cut additional Subtitles
         try:
-            languages = languages[:languages.index("+")]
+            languages = languages.split('+', 1)[0]
```

Comment:
This doesn't really fix anything as the problem is that they stopped using `+` as a delimiter.